explorer/site-config/ncpi-catalog-dug/dev/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const DUG_API_URL =
```

Comment:
It looks like you might only need the origin here, so just `https://search.biodatacatalyst.renci.org`. You have the query string params below (`DUG_API_PARAMS`) and I think you can add the path (`search-api/search_var`) as the `apiPath` in `relatedStudiesEntityConfig`. Let me know if I've missed something here!